gazebo_plugins/CMakeLists.txt

```diff
@@ -81,6 +81,10 @@ if (GAZEBO_VERSION VERSION_GREATER 6.0 OR GAZEBO_VERSION VERSION_EQUAL 6.0)
   catkin_package(  INCLUDE_DIRS include LIBRARIES gazebo_ros_elevator)
 endif()
 
+if (GAZEBO_VERSION VERSION_GREATER 7.3 OR GAZEBO_VERSION VERSION_EQUAL 7.3)
```

Comment:
I think the following is equivalent logic:

~~~
if (NOT GAZEBO_VERSION VERSION_LESS 7.3)
~~~